### ISSUE TITLE
double sided cards broken

### ISSUE THREAD

**Issue by koalasthegreat** [koalasthegreat/AmoeboidBot#4]
double sided card requests will cause the following `KeyError`:

```
Ignoring exception in on_message
Traceback (most recent call last):
  File "/home/pi/.local/lib/python3.7/site-packages/discord/client.py", line 312, in _run_event
    await coro(*args, **kwargs)
  File "/home/pi/bots/AmoeboidBot/bot.py", line 213, in on_message
    raw_cards = scryfall_api.get_cards(card_names)
  File "/home/pi/bots/AmoeboidBot/bot.py", line 180, in get_cards
    image_request = requests.get(raw_card["image_uris"]["normal"])
KeyError: 'image_uris'
```

the JSON response for a double sided card contains a `card_faces` key-value pair, which contains the uris for the card images. These will need to be stitched into one image in the embed, or implemented in some other way

**Comment by koalasthegreat** [comment]
This has been partially resolved in c8e07245e643a085fda7b48ff00467b00d127455, but still doesn't have the full functionality I want. Ideally, both card faces will be shown, and possibly both text boxes as well

**Comment by koalasthegreat** [comment]
This has since been resolved in 0f88101f0910bc59f6789451882de89f80c39c1a
